src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -892,6 +893,23 @@ public static ItemStack harvestPlant(Block block) {
 		return null;
 	}
 
+	public static void harvestFruit(Block fruit) {
+		Location loc = fruit.getLocation();
+		SlimefunItem check = BlockStorage.check(loc);
+		if (check == null) return;
+
+		for (Tree tree : ExoticGarden.getTrees()) {
+			if (check.getID().equalsIgnoreCase(tree.getFruitID())) {
+				BlockStorage.clearBlockInfo(loc);
+				ItemStack fruits = check.getItem();
+				fruit.getWorld().playEffect(loc, Effect.STEP_SOUND, Material.OAK_LEAVES);
+				fruit.getWorld().dropItemNaturally(loc, fruits);
```

Comment:
Don't drop the original, drop a .clone()

Reply:
Boi I literally moved your code >:( It has always worked with this code and still does
Uff I'll change it

Reply:
Something being my code is not a quality indicator XD

Reply:
> Don't drop the original, drop a .clone()

CraftBukkit already creates a copy before spawning an item via `CraftItemStack.asNMSCopy(item)`, so it should be fine as long as if we don't do anything else to this item in our code. But yeah, it's true only for this specific case, and the item normally should be cloned normally.